sematic/api/endpoints/external_resources.py

```diff
@@ -79,7 +82,9 @@ def activate_resource_endpoint(
             "No such resource: {}".format(resource_id), HTTPStatus.NOT_FOUND
         )
     try:
-        activated = record.resource.activate(is_local=False)
+        resource: AbstractExternalResource = record.resource
+        activated = resource.activate(is_local=False)
+        resource.track_usage(user=user, event="activated")
```

Comment:
We probably want a config via which people can "opt out" of tracking, even for "EE" stuff. Paying customers may not want such tracking.

Reply:
Yes I agree. It somewhat defeats the purpose of identifying non-authorized usage, but it's unavoidable.